addons/project/wizard/project_task_convert_to_subtask.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+class ProjectTaskConvertToSubtask(models.TransientModel):
+    _name = "project.task.convert.to.subtask"
+    _description = 'Project Task Convert Wizard'
+
+    @api.model
+    def _default_parent_id(self):
+        return self.env['project.task'].search([('child_ids', '=', self._context.get('active_id'))])
```

Comment:
I think it would be better for performance to get the parent_id through the task_id